.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# SPDX-FileCopyrightText: 2024 Serpent OS Developers
+# SPDX-License-Identifier: CC0-1.0
+
+* @der-eismann @livingsilver94
```

Comment:
Happy to add @GZGavinZhao here, if you want to be bothered with PRs :slightly_smiling_face:

Reply:
Yes, I'm okay with that.